packages/styles/scss/components/feature-card-block-large/_feature-card-block-large.scss

```diff
@@ -34,6 +34,8 @@ $fcb-breakpoint-up--lg: map-get(
     .#{$prefix}--card {
       background: none;
       height: 100%;
+      outline-width: 3px;
+      outline-offset: 1px;
```

Comment:
Looks good @IgnacioBecerra. Small thing, can we put these styles into `:focus` to remain consistent with other component focus states.

Reply:
Updated!